monitoring/opencensus/requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pytest==6.2.1
```

Comment:
I see `flask` here and `Flask` in the other file. Are they different packages? 

`requirements-test.txt` should have the additional requirements for the tests (that aren't needed to run the sample).  `pytest` is usually enough.

Reply:
I've removed this from here, I think it was a mistake - thank you.